pom.xml

```diff
@@ -57,7 +57,7 @@
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter</artifactId>
-            <version>5.6.0</version>
+            <version>5.7.2</version>
```

Comment:
Is this just the latest version? Or is it supposed to align with a version cached in a stack image?   (If so I wonder how we'd keep the two in sync?)

Reply:
I updated the versions in the stack PR that is currently pending review ... so this aligns with those changes.